Source/WebCore/platform/ScrollTypes.cpp

```diff
@@ -216,4 +216,13 @@ TextStream& operator<<(TextStream& ts, ScrollSnapPointSelectionMethod option)
     return ts;
 }
 
+bool defaultScrollbarColoredEnabled()
```

Comment:
This should be declared and implemented in `WebPreferencesDefaultValues.h`/`.cpp` in WebKit.

Reply:
Fixed.